src/System.Data.SqlClient/tests/ManualTests/SQL/TcpDefaultForAzureTest/TcpDefaultForAzureTest.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Diagnostics;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
```

Comment:
Since these tests don't use any user-provided connection strings, they should probably be moved to the FunctionalTests folder.